scripts/lockfile-generators/prefetch-all.sh

```
@@ -137,6 +143,9 @@ done
 [[ -z "$COMPONENT_DIR" ]] && error_exit "--component-dir is required."
 [[ -d "$COMPONENT_DIR" ]] || error_exit "Component directory not found: $COMPONENT_DIR"
 
+export CACHI2_OUT_DIR="cachi2/output/$(python3 -c "import hashlib; print(hashlib.md5('$COMPONENT_DIR'.encode()).hexdigest())")"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)